src/concrete/ml/onnx/convert.py

```diff
@@ -158,15 +163,21 @@ def get_equivalent_numpy_forward_from_torch(
 
 def get_equivalent_numpy_forward_from_onnx(
     onnx_model: onnx.ModelProto,
-    check_model: bool = True,
+    q_x: numpy.ndarray = None,
+    check_model: Optional[bool] = True,
+    use_rounding: Optional[bool] = False,
 ) -> Tuple[Callable[..., Tuple[numpy.ndarray, ...]], onnx.ModelProto]:
     """Get the numpy equivalent forward of the provided ONNX model.
 
     Args:
         onnx_model (onnx.ModelProto): the ONNX model for which to get the equivalent numpy
             forward.
+        q_x (numpy.ndarray): Quantized input used to compute the LSBs to remove if 'use_rounding'.
```

Comment:
question : this is currently used for the workaround (`compute_lsb_to_remove_for_trees`) right ? But will it be necessary to have when the truncate feature will come ? If no, that means we'll have to break the API here since `get_equivalent_numpy_forward_from_onnx` is not private, so maybe there could be another way than adding this new parameter ?